src/information-panel/information-panel.scss

```diff
@@ -187,3 +192,34 @@
     top: $iui-m * -0.5;
   }
 }
+
+/// Labels and inputs aligned using css grid
+@mixin iui-information-body-content {
+  .iui-input-label {
+    font-weight: normal;
+    @include themed {
+      color: t(iui-text-color-muted);
+    }
+  }
+
+  &:not(.iui-inline) {
```

Comment:
When it is applied? I can't find it in demo page.

Reply:
Ah good catch. I was going to toggle this class using JS

Reply:
Added JS